### PR TITLE
Pull things into master

### PR DESCRIPTION
**Pull request by abroekhof**

### CODE REVIEW

**Review by dehavenm — APPROVED** [review]
When I tested it, it was still slightly overshooting stop lines. Other than that, it all appears to do what it's supposed to do.

File: ros/src/tl_detector/light_classification/tl_classifier.py [hunk 1]
```diff
@@ -1,9 +1,206 @@
...
+            ])
+
+
 class TLClassifier(object):
     def __init__(self):
-        #TODO load classifier
-        pass
+        self.model = None
+
+        if rospy.get_param('/launch') == 'styx':
```
Comment (joshchao39) [inline ros/src/tl_detector/light_classification/tl_classifier.py]: I wonder if the site uses the same launch script? If not `get_param` might raise a`KeyError` if `/launch` is not found. Maybe it's better to default to site unless the parameter is available and says `styx`

Comment (abroekhof) [reply ros/src/tl_detector/light_classification/tl_classifier.py]: I defined the parameters in both `launch` files so this should work!

Comment (joshchao39) [reply ros/src/tl_detector/light_classification/tl_classifier.py]: I was worried that they will swap out the launch script for site. Maybe Carla has other ROS nodes that's not included in the launch script here. Anyway I probably worried too much.

Comment (abroekhof) [reply ros/src/tl_detector/light_classification/tl_classifier.py]: That makes sense—I think the key error would be good in that case, kind of acts like a compilation error...

**Pull request merged**